.github/ISSUE_TEMPLATE/Doc_changes.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Documentation Changes
+about: Create a report to help us improve
+
+---
+
+### Document Change needed
+
+A clear description of what changes are needed with the location of the error that needs to be fixed 
+
+Example:
+In the `master` version of the documentation there is an error in section "13.2.1", the offending line in 17 and can be found in
+the file `/pecan/book_source/04_advanced_user_guide/01_R_workflow.Rmd' 
+
+### Information for those wanting to make changes
+
+Please follow the instructions in the [Editing this Book](https://pecanproject.github.io/pecan-documentation/develop/GettingStarted.html) section of the documentation. 
+
+All R-markdown files related to documentation are located within the main directory `/pecan/book_source'
```

Comment:
I would replace this with "All R-markdown files related to documentation are located within the main directory `/pecan/book_source` except for the demos and vignettes which are in `pecan/documentation/tutorials`."